src/main/kotlin/no/nav/syfo/domain/ManglendeMedvirkningVurdering.kt

```diff
@@ -71,3 +72,20 @@ data class Varsel(
     val createdAt: OffsetDateTime,
     val svarfrist: LocalDate,
 )
+
+fun Status.getDokumentTittel(): String = when (this) {
```

Comment:
Holder kanskje med `dokumentTittel` ettersom det nærmest er en property for statusen? 😊
```suggestion
fun Status.dokumentTittel(): String = when (this) {
```